packages/playout-gateway/src/tsrHandler.ts

```diff
@@ -479,31 +477,30 @@ export class TSRHandler {
 				return result
 			})
 		}
+		const devices: {
```

Comment:
Perhaps this could be made into a `Map<string, DeviceOptionsAny>`?
It will require checking/updating all usages, so not essential to do right now, but should be done at some point to better handle `undefined` values